pkgs/development/python-modules/py-machineid/default.nix

```diff
@@ -0,0 +1,33 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "py-machineid";
+  version = "0.6.0";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-AMONhSHUKaRTm92SlnI02yihorSyYwYrNRygAjMuYz8=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  pythonImportsCheck = [ "machineid" ];
+
+  # Tests are not present in Pypi archive
+  doCheck = false;
+
+  meta = {
+    description = "Get the unique machine ID of any host (without admin privileges";
+    homepage = "https://pypi.org/project/py-machineid/";
```

Comment:
How about citing the GitHub repository?
https://github.com/keygen-sh/py-machineid

Reply:
Sure !